src/pages/Home.tsx

```diff
@@ -21,58 +21,23 @@ export default function Home({
   onSeedChange,
   groups,
   onGroupsChange,
-}: HomeProps) {
+}: Props) {
   const rng = seedrandom(seed)
```

Comment:
How about calling it by it's name? 
```suggestion
  const randomSeed = seedrandom(seed)
```

Reply:
The return of `seedrandom("my-sweet-seed")` is the random number generator not the seed

Reply:
Maybe then `randomNumberGenerator`?